tracer/src/Datadog.Trace/Agent/DiscoveryService/DiscoveryService.cs

```diff
@@ -82,6 +82,24 @@ public static DiscoveryService Create(ImmutableExporterSettings exporterSettings
                     () => new MinimalAgentHeaderHelper(),
                     uri => uri));
 
+        public static DiscoveryService Create(
```

Comment:
Can we combine this with the Create function above? Maybe move the default values for the retries etc to this method, and get rid of the second constructor?

Reply:
I removed the .ctor, but I did not combine both `Create` functions, there's a TimeSpan parameter that we cannot have as a default value (but maybe as an optional parameter???)